src/SubmitPage/SubmitForm.js

```diff
@@ -302,31 +366,43 @@ class SubmitForm extends React.Component {
               />
             </Form.Group>
 
-            <Form.Group controlId="formBasicServiceType">
-              <Form.Label>{translations.store_category}</Form.Label>
-              <Form.Control
-                as="select"
-                value={formData[STORE_CATEGORY]}
-                onChange={e => this.onChangeInput(e, STORE_CATEGORY)}
-              >
-                <option value={STORE_CATEGORIES.GROCERY}>
-                  {translations.grocery}
-                </option>
-                <option value={STORE_CATEGORIES.RESTAURANT}>
-                  {translations.restaurant}
-                </option>
-                <option value={STORE_CATEGORIES.ATM}>{translations.atm}</option>
-                <option value={STORE_CATEGORIES.CLINIC}>
-                  {translations.clinic}
-                </option>
-                <option value={STORE_CATEGORIES.PHARMACY}>
-                  {translations.pharmacy}
-                </option>
-                <option value={STORE_CATEGORIES.OTHER}>
-                  {translations.other}
-                </option>
-              </Form.Control>
-            </Form.Group>
+            {enableTags && (
+              <AvailabilityTags
+                tags={this.state.tags}
+                setTags={this.setTags}
+                translations={translations}
+              />
+            )}
+
+            {!enableTags && (
+              <Form.Group controlId="formBasicServiceType">
```

Comment:
i'm assuming hiding store category in the case of availability tags is by design?

Reply:
Yep. Sanjeev added a [comment](https://github.com/covid-maps/covid-maps/pull/262#issuecomment-615946665) about this change